### PR TITLE
#13 - Fix requested changes

### LINKED ISSUE #13: Fix comments on Pull-Request
https://github.com/ading-be/MTX.Conversion.ZebraWrap/pull/8#pullrequestreview-399762768

### PR DESCRIPTION
**Pull request by HavermansStef**
- Closes #13

### CODE REVIEW

File: MTX.Conversion.ZebraWrap.App/Configuration/ConfigurationProvider.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    {
+        #region Public Members
+
+        private readonly AppSettings _appSettings;
+        
+        #endregion
+
+        #region Constructor
+
+        public ConfigurationProvider()
```
Comment (BartVanHoutte-ADING) [inline MTX.Conversion.ZebraWrap.App/Configuration/ConfigurationProvider.cs:15]: remove empty constructor

To much refactor? No logic

File: MTX.Conversion.ZebraWrap.App/Program.cs [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            var basePath = FindBasePathFor(appSettingsName);
+            var builder = new ConfigurationBuilder()
+                .SetBasePath(basePath)
+                .AddJsonFile(appSettingsName);
+
+            Directory.SetCurrentDirectory(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location));
+            return builder.Build();
+        }
+
+        private static string FindBasePathFor(string fileName)
```
Comment (BartVanHoutte-ADING) [inline MTX.Conversion.ZebraWrap.App/Program.cs:50]: Move to configurationprovider (for testing purposes)

File: MTX.Conversion.ZebraWrap.Contracts/Helpers/WrapHelper.cs [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+using System.Linq;
+using MTX.Conversion.ZebraWrap.Contracts.Constants;
+
+namespace MTX.Conversion.ZebraWrap.Contracts.Helpers
+{
+    public static class WrapHelper
+    {
+        public static void AddField(this List<FieldDto> fields, string fieldValue, string fieldName, int addedFields)
+        {
+            fields.Add(new FieldDto
```
Comment (BartVanHoutte-ADING) [inline MTX.Conversion.ZebraWrap.Contracts/Helpers/WrapHelper.cs:11]: var addedFields = label.Fields.Count(x => x.Name.StartsWith($'{fieldName},')) + 1;

File: MTX.Conversion.ZebraWrap.Contracts/Helpers/WrapHelper.cs [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+            {
+                Value = fieldValue,
+                Name = $"{fieldName},{addedFields}",
+                Id = fields.Max(x => x.Id) + addedFields,
```
Comment (BartVanHoutte-ADING) [inline MTX.Conversion.ZebraWrap.Contracts/Helpers/WrapHelper.cs:15]: label.Field.Max(x => x.Id) + 1

Comment (HavermansStef) [reply MTX.Conversion.ZebraWrap.Contracts/Helpers/WrapHelper.cs:15]: Fixed.

File: MTX.Conversion.ZebraWrap.Contracts/Helpers/WrapHelper.cs (hunk 3 above)
Comment (HavermansStef) [reply MTX.Conversion.ZebraWrap.Contracts/Helpers/WrapHelper.cs:11]: Fixed.

**Pull request merged**